src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -361,6 +362,27 @@ export class AccessibleView extends Disposable {
 		return symbols.length ? symbols : undefined;
 	}
 
+	configureKeybindings(): void {
+		const items = this._currentProvider?.options?.configureKeybindingItems;
+		if (!items) {
+			return;
+		}
+		const quickPick: IQuickPick<IQuickPickItem> = this._quickInputService.createQuickPick();
+		this._register(quickPick);
```

Comment:
You should dispose the quickpick when you're done with it, and the listeners below should also be disposed

Reply:
I think I do on lines 381 and 383. The listeners are registered to the quick pick I believe?